include/shell/shell.h

```diff
@@ -660,6 +635,16 @@ int shell_cmd_precheck(const struct shell *shell,
 void shell_print_stream(const void *user_ctx, const char *data,
 			size_t data_len);
 
+/**
+ * @brief Prints the current command help.
+ *
+ * Function will print a help string with: the currently entered command
+ * and subcommands (if they exist).
+ *
+ * @param[in] shell      Pointer to the shell instance.
+ */
+void shell_help_print(const struct shell *shell);
```

Comment:
Any particular reason not to call it shell_help?

Reply:
no, I will rename it.

Reply:
done